app/src/main/java/ch/epfl/sdp/appart/database/MockDatabaseService.java

```
@@ -162,6 +162,13 @@ public MockDatabaseService() {
         return result;
     }
 
+    @NonNull
+    @Override
+    public CompletableFuture<Boolean> deleteImage(String imagePathAndName) {
+        // TODO: implementation soon
```

We should throw an exception in unimplemented method if you agree.